core/arch/arm/arm.mk

```diff
@@ -210,12 +210,25 @@ supported-ta-targets += ta_arm64
 endif
 endif
 
+ifeq (,$(supported-sp-targets))
+supported-sp-targets = sp_arm32
```

Comment:
We are planning to add support for AArch32 platforms in CQ1'22 according to [our roadmap](https://developer.trustedfirmware.org/w/trusted-services/roadmap/). This feature mainly targets low-end platforms.
The SP dev kit now contains both AArch32 and AArch64 support despite we don't have the AArch32 SPMC in place.
As the SP dev kit is based on the TA dev kit where the AArch32 support is already available it was easier to keep this part of the system because we are planning to use it in the near future.

Reply:
OK, thanks.